manager/src/test/java/com/redhat/service/smartevents/manager/utils/TestUtils.java

```diff
@@ -44,6 +46,26 @@ public static Response getBridges() {
                 .get(APIConstants.USER_API_BASE_PATH);
     }
 
+    public static Response getBridgesFilterByName(String name) {
+        return jsonRequest()
+                .get(APIConstants.USER_API_BASE_PATH + "?name=" + name);
+    }
+
+    public static Response getBridgesFilterByStatus(ManagedResourceStatus... status) {
+        String queryString = Arrays.stream(status).map(s -> "status=" + s).collect(Collectors.joining("&"));
+        return jsonRequest().get(APIConstants.USER_API_BASE_PATH + "?" + queryString);
+    }
+
+    public static Response getBridgesFilterByStatusWithAnyValue(String... status) {
+        String queryString = Arrays.stream(status).map(s -> "status=" + s).collect(Collectors.joining("&"));
+        return jsonRequest().get(APIConstants.USER_API_BASE_PATH + "?" + queryString);
+    }
+
+    public static Response getBridgesFilterByNameAndStatus(String name, ManagedResourceStatus... status) {
+        String queryString = Arrays.stream(status).map(s -> "status=" + s).collect(Collectors.joining("&"));
+        return jsonRequest().get(APIConstants.USER_API_BASE_PATH + "?name=" + name + "&" + queryString);
+    }
```

Comment:
Hi @jomarko I'll answer your other questions soon; but for this one: A HTTP Query String is formatted `?key1=value1&key2=value2` etc. If you want a key to have multiple values you can specify the same key multiple times. So, since @kelvah wanted to have a collection of possible Statuses I build the Query String as `?status=ready&status=accepted` etc.

I [commented](https://issues.redhat.com/browse/MGDOBR-658?focusedCommentId=20248471&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-20248471) this in the JIRA and @kelvah seemed OK with it. I have tested generation of the type script bindings for the UI from the new `openapi.yaml` file and it builds a similar Query String too.

Reply:
@manstis ok, thank you!